frontend/utils.mjs

```diff
@@ -75,10 +81,16 @@ export function copyIndexHtml(from = 'src/index.html', to = 'dist/index.html', e
                             } 
                         } 
                     }
-                    window.ESBUILD_LOAD_SCRIPT("${entry}.js?t=" + new Date().valueOf())
+                    window.ESBUILD_LOAD_SCRIPT(${
+                        entrypoints.length > 0
```

Comment:
Nit: Let's extract this to a var and explicitly mention there can only be 1 js/css endpoint if passed an array. IT's not really obvious as-is